docs/usage/self-hosted-configuration.md

```diff
@@ -175,6 +175,7 @@ Supported tools for dynamic install are:
 - `pnpm`
 - `poetry`
 - `python`
```

Comment:
Will `semantic-release` work properly if the PR title has an emoji like 🎉?

Reply:
Nevermind, I see `semantic-release` handles emojis just fine. 🎉 😏

Reply:
😜